content/en/docs/event-sources/okta.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Okta Events
+weight: 2
+---
+
+The Falco [Okta](https://github.com/falcosecurity/plugins/blob/master/plugins/okta/README.md) plugin can read [Okta](https://www.okta.com/) logs and emit events for each Okta log entry.
+
+Falco also distributes out-of-the-box [rules](https://github.com/falcosecurity/falco/blob/master/rules/okta_rules.yaml) that can be used to identify interesting/suspicious/notable events in Okta logs, including:
+
+* Creating a new OKTA user account
+* Detecting a locked-out user
+* Assigning admin permissions to an okta user
+
+# Configuration
+
+The [README](https://github.com/falcosecurity/plugins/blob/master/plugins/okta/README.md#settings) page has full details on how to configure the plugin. This simply involves providing the organization/api token as a part of init params. These can be added to `falco.yaml` under the `plugins` [configuration key](https://falco.org/docs/configuration/) key.
+
+The plugin does not use any open params configuration.
+
+# Sample Output
+
+For a dummy rule like the following:
```

Comment:
For example:

Reply:
I want to make it clear that the output depends on the rule used to filter the events, so I ended up with "For example, when using a dummy rule like the following:".